rest-endpoint/src/main/java/com/cognifide/aet/rest/MetadataServlet.java

```diff
@@ -74,6 +79,7 @@ protected void process(DBKey dbKey, HttpServletRequest req, HttpServletResponse
     String suiteName = req.getParameter(Helper.SUITE_PARAM);
     String suiteVersion = req.getParameter(Helper.VERSION_PARAM);
     String formatted = req.getParameter(FORMATTED_PARAM);
+    resp.setHeader(APP_VERSION_HEADER, bundleVersionProvider.getBundleVersion());
```

Comment:
I think it's fine because this is some kind of "meta" information. But maybe we should prefix the custom header name with `X-` (`X-Application-Version`), WDYT?
CC @Skejven

Reply:
`X-` prefix is a good idea. Sending it in the header is fine.